pysces/panel.py

```diff
@@ -46,6 +50,15 @@ def _update(self, Uinfty=(1,0)):
         self._vortices = Vortices(xvort)
         self._influence_matrix = None
 
+        # Uncomment the following lines to see the vortex and collocation
```

Comment:
The import statement and the lines beginning with "#Uncomment the following
lines" can be removed.  I put those in during testing so I could better
understand what the code was doing.  I agree on principle that matplotlib
shouldn't be a dependency of the main pysces code.

On Sun, Nov 22, 2015 at 8:01 AM, Clancy Rowley <notifications@github.com>
wrote:

> In pysces/panel.py
> <https://github.com/cwrowley/pysces/pull/2#discussion_r45558523>:
>
> > @@ -46,6 +50,15 @@ def _update(self, Uinfty=(1,0)):
> >          self._vortices = Vortices(xvort)
> >          self._influence_matrix = None
> >
> > +        # Uncomment the following lines to see the vortex and collocation
>
> Looks to me as though this is the only place matplotlib is needed. Could
> put "import matplotlib.pyplot as plt" here (commented out) if you want to
> leave in this plotting stuff, and then we still wouldn't depend on
> matplotlib.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cwrowley/pysces/pull/2/files#r45558523>.
>